src/tunnel.cc

```diff
@@ -951,6 +966,9 @@ TunnelStateData::tunnelEstablishmentDone(Http::TunnelerAnswer &answer)
 
     al->cache.code.update(LOG_TCP_TUNNEL);
 
+    // XXX: al->hier.peer_reply_status (i.e. *status_ptr) should not be (re)set
```

Comment:
I guess you meant  al->http.code:
```suggestion
    // XXX: al->http.code (i.e. *status_ptr) should not be (re)set
```